account_fiscal_position_partner_type/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Valentín Vinagre <valentin.vinagre@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Fiscal Position Partner Type",
+    "version": "13.0.0.0.1",
+    "category": "Accounting & Finance",
+    "website": "https://github.com/OCA/account-fiscal-rule",
+    "author": "Sygel Technology," "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "development_status": "Production/Stable",
```

Comment:
This can be `Alpha` or `Beta`

Reply:
If it meets the requirements (good test coverage), it can directly be this status.

Reply:
This module has no tests

Reply:
Then it can't be indeed

Reply:
Hi @sergiocorato ,
The pr is in wip and there is a point that is to do tests that is not marked, I just uploaded the tests. If still the state has to be beta, I change it, I have no problem.

Reply:
Now is ok, thanks